tests/miral/org_kde_kwin_server_decoration.c

```diff
@@ -0,0 +1,71 @@
+/* Generated by wayland-scanner 1.16.0 */
```

Comment:
I get this is a test, but even so why is there a generated C file?

Reply:
Simpler than ensuring `wayland-scanner` is available and generating it every time. There's a `.h` too.

Reply:
Sounds good